### ISSUE TITLE
Positron outputs panel is overwhelming to new users

### ISSUE THREAD

**Issue by jthomasmock** [posit-dev/positron#5063]
#### Positron and OS details:

Positron Version: 2024.11.0 (Universal) build 49
Code - OSS Version: 1.93.0
Commit: 10183fa8d9a8495b5d52cafd252cfa5a44b05599
Date: 2024-10-15T02:46:25.482Z
Electron: 30.4.0
Chromium: 124.0.6367.243
Node.js: 20.15.1
V8: 12.4.254.20-electron.0
OS: Darwin arm64 24.0.0

## Describe the issue:

There is a wealth of debugging and logging information in the Output panel, however there is an overwhelming amount of information/options:

![Image](https://github.com/user-attachments/assets/e5d1cc0e-1419-46a2-80ab-2866d63bfdd1)

## Steps to reproduce the issue:

Open the Output panel.

## Expected or desired behavior:

We should split up, add context, insert breaks or otherwise guide the user towards the most important processes for debugging. This may be splitting out the Positron-specific outputs into their own section or otherwise guiding the user towards the most useful information.

**Comment by testlabauto** [comment]
Moving to triage for prioritization.

**Comment by DavisVaughan** [comment]
I tend to disagree on having a "positron specific" section. I think we have discussed in the past that we should not really be differentiating between vs code features and positron features in this output pane - i.e. they are _all_ positron features!

To that end I think we wanted to put a little more effort towards removing the `Positron` prefix from a lot of these channel names, I think @juliasilge originally noted this. Instead we'd use the language name as the leading identifier to group things, like `R:` or `R 4.4.1:`, kind of like how its mostly done for Python already. Candidates for this are:
- `Positron Language Server: R 4.4.1 (console)`
- `Positron R Extension`
- `Positron R Profiler`

**Comment by jmcphers** [comment]
Ideas from @jthomasmock and others in triage:

- have a peer of `Output` that just shows Positron stuff
- better grouping in this menu so you can find Positron stuff
- task-specific problem reporting entry point

**Comment by lionel-** [comment]
Related idea to help our users report all relevant log outputs:

- Extensions (positron-r, positron-python) would implement a command to open all relevant output channels as read-only _editors_. This allows them to review the contents before posting without forgetting one of the channel.

- Another command would save these channels to a zip file ready to be attached to an issue.

Since the number of our log channels tend to go up, this would make it easier for our users to report them, and it would ensure our documentation at https://positron.posit.co/troubleshooting.html#python-and-r-logs always stays up to date even when a channel is added, renamed, or removed.

**Comment by jmcphers** [comment]
Sufficiently addressed by planned work to overhaul the Console info pane with links to the right output channels et al.
